resources/authgear/templates/en/web/authflowv2/enter_oob_otp.html

```diff
@@ -1,7 +1,77 @@
 {{ template "authflowv2/__page_frame.html" . }}
 {{ define "page-content" }}
 
-<h1>Enter OOB OTP</h1>
-TO BE IMPLEMENTED
+  {{ $has_alternatives := false }}
```

Comment:
Is it possible to use the same html template in both handler?
e.g. We only have a enter_oob_otp.html, and both whatsapp_otp and enter_oob_otp handler renders this template.